lib/controllers/onepiece.js

```diff
@@ -0,0 +1,48 @@
+const { Router } = require('express');
+const { getAll, getById } = require('../models/Onepiece');
+const Character = require('../models/Onepiece');
+const pool = require('../utils/pool');
+
+module.exports = Router()
+  .post('/', async (req, res) => {
+    const character = await Character.insert({
+      name: req.body.name,
+      crew: req.body.crew,
+    });
+    res.json(character);
+  })
+  .get('/', async (req, res) => {
+    try {
+      const allCharacters = await getAll();
+      res.json(allCharacters);
+    } catch (error) {
+      error;
```

Comment:
We will cover error handle more in depth, but for now we should pass this error along to our error handling middleware with `next(error)`